### ISSUE TITLE
Require IE11+ for platform

### ISSUE THREAD

**Issue by reefdog** [documentcloud/documentcloud#437]
We already decided to end support for IE9 and older (see conversation starting [here](https://github.com/documentcloud/documentcloud/pull/418#issuecomment-257365592)). Requiring IE11 would free us up even more, letting us shed some shivs and polyfills and start using flexbox.

IE10 was **0.4%** of our traffic over the past year. (That's actually less than our IE8 traffic.) [Microsoft no longer supports it.](https://www.microsoft.com/en-us/WindowsForBusiness/End-of-IE-support) Adding it to our unsupported list raises our "unsupported IE traffic" from 1.0% to 1.4%. I think this is all reasonable.

**Comment by reefdog** [comment]
I should note that this is only for the platform, not the embeds, where we could be a little more generous. Although of course any iframed embeds would de facto adopt the platform's requirements.

**Comment by knowtheory** [comment]
always helpful to refer back to https://analytics.usa.gov/ too.

Although frankly at this point i'm fine moving forward.

**Comment by reefdog** [comment]
Yup, that's a useful proxy for our embeds particularly since it represents "the wider web" better than our platform numbers. They've got IE less than 11 at ~2.1%.
